lightning/src/ln/channelmanager.rs

```diff
@@ -1672,7 +1672,9 @@ macro_rules! handle_error {
 					if let Some((channel_id, user_channel_id)) = chan_id {
 						$self.pending_events.lock().unwrap().push_back((events::Event::ChannelClosed {
 							channel_id, user_channel_id,
-							reason: ClosureReason::ProcessingError { err: err.err.clone() }
+							reason: ClosureReason::ProcessingError { err: err.err.clone() },
+							counterparty_node_id: Some($counterparty_node_id),
+							channel_capacity: None,
```

Comment:
Hmm. Going to be icky to try pass the capacity through to here, but I feel like it's something we should figure out how to do nicely. We should always populate these new fields when we create the event from now on.

Reply:
Will look into this